src/config/dynamic-theme-fixes.config

```diff
@@ -18094,6 +18103,12 @@ body,
 .topbar .icon-site-switcher-bubble {
     background-repeat: no-repeat !important;
 }
+div.h100.bar-lg.p-bs-wrapper:has(a[href="/questions"]) {
```

Comment:
FYI don't do multiple sites fixes in a PR, unless indicated in the commit title.

Reply:
Yeah, the PR title was accidental (I created a PR and then remembered that I saw a very similar bug elsewhere). Also, a proper fix would be to implement support for lowering expressions with `background` set to `linear-gradient` dependent on some `var`.

Reply:
Unfortunately that's not possible. The browser doesn't show a `var(...)` value for the `background` property. The browsers itself doesn't know how to resolve it at a CSS level.

Reply:
stackoverflow.com has the following CSS (for example):
```css
background: linear-gradient(0deg, var(--orange-100) 30%, var(--white))
```
So we might want to lower that and substitute our vars in place of these?

Reply:
> So we might want to lower that and substitute our vars in place of these?

Yeah that actually should be the case, dark reader should be capable of modifying that CSS rule.